File: python/get_childfolders.py
import http.client

conn = http.client.HTTPSConnection("graph.microsoft.com")

conn.request("GET", "/v1.0/me/mailFolders/%7Bid%7D/childFolders")

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: ruby/get_childfolders.rb
require 'uri'
require 'net/http'

url = URI("https://graph.microsoft.com/v1.0/me/contactFolders/%7Bid%7D/childFolders")

http = Net::HTTP.new(url.host, url.port)
http.use_ssl = true
http.verify_mode = OpenSSL::SSL::VERIFY_NONE

request = Net::HTTP::Get.new(url)

response = http.request(request)
puts response.read_body
